fearless/Modules/AccountImport/AccountImportViewLayout.swift

```diff
@@ -44,6 +44,7 @@ final class AccountImportViewLayout: UIView {
         view.actionControl.contentView.subtitleLabelView.textColor = R.color.colorWhite()
         view.actionControl.contentView.subtitleLabelView.font = .p1Paragraph
         view.actionControl.contentView.subtitleLabelView.numberOfLines = 1
+        view.actionControl.contentView.subtitleImageView.isHidden = true
```

Comment:
please ensure that subtitleImageView should be always hidden

Reply:
Yes, he unused